### ISSUE TITLE
Missing symlinks to bootstrap profiles breaks bootstrap process as pillar cannot be loaded

### ISSUE THREAD

**Issue by uvsmtid** [uvsmtid/common-salt-states#6]
Salt configuration file (normally `/etc/salt/master`) in bootstrap package currently contains bootstrap configuration:

```
# ...
this_system_keys:
    # ...
    load_bootstrap_target_envs:
        automate-env: ~
# ...
```

However, necessary symlinks under `pillars/bootstrap/profiles` directory are not created during execution of bootstrap script:

```
DEBUG:root:command line: "salt-call" "--out" "json" "--log-level" "debug" "state.sls" "common.source_symlinks" "test=False" "pillar={ 'bootstrap_mode': 'initial-online-node' }"
DEBUG:root:stdout_file_path: /tmp/tmpvDa_hb
DEBUG:root:stdin_string: None
DEBUG:root:before wait
[DEBUG   ] Reading configuration from /etc/salt/minion
[DEBUG   ] Configuration file path: /etc/salt/minion
[DEBUG   ] Reading configuration from /etc/salt/minion
[DEBUG   ] Decrypting the current master AES key
[DEBUG   ] Loaded minion key: /etc/salt/pki/minion/minion.pem
[DEBUG   ] Loaded minion key: /etc/salt/pki/minion/minion.pem
[DEBUG   ] Mako not available
[DEBUG   ] LazyLoaded .returner
[DEBUG   ] Decrypting the current master AES key
[DEBUG   ] Loaded minion key: /etc/salt/pki/minion/minion.pem
DEBUG:root:after wait
{
    "local": [
        "Pillar failed to render with the following messages:", 
        "Specified SLS 'bootstrap.profiles.automate-env' in environment 'base' is not available on the salt master"
    ]
}
```

**Comment by uvsmtid** [comment]
At the moment bootstrapped setup is configured to be able to generate bootstrap package for itself.
This is primarily done to build the package in self-configured Salt (in CI). However, it does not bootstrap itself in the CI (it only updates Salt configuration in CI).

So, the default could be changed to use none bootstrap profiles for "bootstrap time" and configured bootstrap profiles for "CI time".

**Comment by uvsmtid** [comment]
Related to #8.

**Comment by uvsmtid** [comment]
The issue was fixed by commit d92f2ff776c1a922171c917f82d9c765cbfe9430.

Instead of using `None` (or empty list) for bootstrap profiles, the bootstrap script simply creates as many symlinks as required to satisfy list in `load_bootstrap_target_envs` (although all of them point to the same `bootstrap-target` pillars - they all contain the same pillars).
